cypress-tests/cypress/support/utils/version.js

```diff
@@ -149,8 +143,9 @@ export const verifyVersionAfterPreview = (currentVersion) => {
     .invoke("removeAttr", "target")
     .click();
   cy.url().should("include", "/home");
-  verifyComponent("button1");
-  cy.go("back");
-  cy.waitForAppLoad()
-  cy.contains(currentVersion);
+  cy.wait(2000);
+  cy.get('[data-cy^="draggable-widget-table"]').should('be.visible');
+  cy.url().should("include", `version=${currentVersion}`);
+  cy.get('[data-cy="viewer-page-logo"]').click();
+  cy.wait(8000);
```

Comment:
@ajith-k-v could you please let me know why we using 8second wait in the end of the spec?